### ISSUE TITLE
Connection info shows N/A on some systems

### ISSUE THREAD

**Issue by jwijenbergh** [eduvpn/python-eduvpn-client#493]
Reported by @aretsmarvin

# Installation

How did you install the eduVPN client?

Pip

# Version

What version of the client you are running? Try your package manager or otherwise `$ eduvpn-client -v`.
3.0.0

# OS/Distribution

What operating system/distribution and version you are running?

Endeavour os

# Logs

```
2022-06-14 16:50:52,622 - run-periodically - WARNING - eduvpn.ui.stats - stats.py:89 - Network Stats: rx_bytes, failed to get interface
2022-06-14 16:50:52,622 - run-periodically - WARNING - eduvpn.ui.stats - stats.py:89 - Network Stats: tx_bytes, failed to get interface
```

The ip fails to be returned because the interface is not found

**Comment by jwijenbergh** [comment]
@aretsmarvin does this still happen with #494?

**Comment by aretsmarvin** [comment]
@jwijenbergh yes. #494 seems to work :)

The arrow back > select profile issue is back/still exists.

**Comment by jwijenbergh** [comment]
> @jwijenbergh yes. #494 seems to work :)
> 
> The arrow back > select profile issue is back/still exists.

Cool, the arrow back issue is fixed with #491 which is not merged yet. Thanks for testing!

**Comment by jwijenbergh** [comment]
Arrow back issue is #465
